format/src/main/thrift/schema.thrift

```diff
@@ -124,6 +124,7 @@ struct TableSchema{
 	1: required string table_id;  // ID used to
 	2: required list<ColumnSchema> table_columns; // Columns in the table
 	3: required SchemaEvolution schema_evolution; // History of schema evolution of this table
+	4: optional i32 block_size
```

Comment:
@manishgupta88 
1. Now we put all columnProperties in format.ColumnSchema, and all ColumnSchema would be stored into format.TableSchema, why not table property(block.size) it self can not stored into format.TableSchema?  I think its ok.
2. We must using file to store this info, if not, when restart the message would lost. Further more it is not reasonable to use one new file who only store this info.